megamek/src/megamek/common/verifier/TestEntity.java

```diff
@@ -1933,8 +1933,12 @@ public String printTechLevel() {
                 + getEntity().getYear() + ")\n";
     }
 
+    /**
+     * @return The total additional weight due to critical slot armoring. Does not include the weight of the equipment
+     * itself.
+     */
     public double getArmoredComponentWeight() {
-        return 0.0;
+        return 0;
```

Comment:
Internally I know it'll convert, but why removal of the `.0`?

Reply:
what is 0.0 good for? Zero is zero, doesnt get better as 0.0. Why have letters that are meaningless and not particularly informative?